tests/test_status_codes.py

```diff
@@ -0,0 +1,28 @@
+import http
+import sys
+from typing import Tuple
+
+import pytest
+
+from falcon import status_codes
+
+
+class TestStatusCodes:
+    @pytest.mark.skipif(
+        sys.version_info < (3, 13), reason='Outdated http statuses definitions'
+    )
+    @pytest.mark.parametrize('status', status_codes.__all__)
+    def test_statuses_are_in_compliance_with_http_from_python313(self, status):
+        status_code, message = self._status_code_and_message(status)
+        if status_code >= 700:
+            pytest.skip('Codes above 700 are not defined in http package')
+        http_status = http.HTTPStatus(status_code)
+        if status_code in [413, 418, 422]:
+            assert http_status.phrase != message
```

Comment:
I don't think we should write negative tests like that since they require active maintenance. But we can improve that in #2276.